blockchain/chainio.go

```diff
@@ -1221,6 +1221,25 @@ func (b *BlockChain) initChainState() error {
 			return err
 		}
 
+		// As a final consistency check, we'll run through all the
+		// nodes which are ancestors of the current chain tip, and mark
```

Comment:
Is it worth checking the opposite for orphaned blocks?

Reply:
Orphaned blocks may have never been validated and, as such, there are no guarantees as to their expected validity.